src/cata_tiles.h

```diff
@@ -319,8 +343,22 @@ class cata_tiles
         /** Variables */
         SDL_Renderer *renderer;
         tile_map tile_values;
+        tile_map shadow_tile_values;
```

Comment:
These new member variables should get the same treatment as the new methods, default to private instead of protected.